classes/static/js/toolbar.js

```diff
@@ -0,0 +1,24 @@
+jQuery( document ).ready(function() {
+	jQuery("body").append('<div id="autoptimize-loader-toolbar"></div>');
```

Comment:
That does not make that the element `div `will be created each time the event is fired?